crates/compiler/build/src/link.rs

```diff
@@ -457,6 +434,7 @@ pub fn rebuild_host(
     };
 
     let host_dest = if matches!(target.architecture(), Architecture::Wasm32) {
+        // TODO verify this is corect, how do we do get here with OptLevel::Development
```

Comment:
TODO -- investigate

Reply:
I think we should leave this for a follow-up PR. This is a very niche use case for Roc and AFAIK there are no platforms using this in the wild.